_data/people_list.yml

```
@@ -271,6 +271,13 @@
   repos:
   - Nemocas/AbstractAlgebra.jl
   - oscar-system/Oscar.jl
+- name: Marina Garrote
+  email: marina.garrote@mis.mpg.de
+  github: garrote
+  status: active
+  comment: Co-author of commit ad7e39a9abb981d0873a67477d50d3664b720e00
```

@marinagarrote Can you verify this is how you would like to have your details displayed?
```suggestion
- name: Marina Garrote
  email: marina.garrote@upf.edu
  github: marinagarrote
  status: active
  comment: Co-author of commit ad7e39a9abb981d0873a67477d50d3664b720e00
```

---

@marinagarrote I will now take the changes that @antonydellavecchia proposed above and merge, so that this updated information becomes visible on the Oscar website. If you wish to update/modify, please get in touch. Thank you!